core/fortran-example/tutorial/05_parallel_for/parallel_for_example.cpp

```diff
@@ -0,0 +1,213 @@
+/****************************************************************************
+ * Copyright (c) 2018-2019 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cabana_Core.hpp>
+
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Random.hpp>
+
+#include <iostream>
+
+#include "veclen.h"
+
+//---------------------------------------------------------------------------//
+// parallel for example.
+//---------------------------------------------------------------------------//
+
+//This is the coresponding struct_of_array defined by DataTypes
+struct local_data_struct_t {     
+  double d0[veclen];     
+  double d1[veclen];     
+};
+extern "C" {
+  void c_kokkos_initlize( void );
+  void c_kokkos_finalize( void );
+  void parallelForExample();
+  __host__ __device__ void initialization(local_data_struct_t*,int);      
+  __host__ __device__ void kernel_1(local_data_struct_t*,int,int);      
+  __host__ __device__ void kernel_2(local_data_struct_t*,int,int,int,int);      
+}
+
+
+void parallelForExample()
+{
+    /*
+      In previous examples we have demonstrated using the Slice directly with
```

Comment:
Instead of copy-pasting these comments from the C++ tutorial let's update them to reflect what this Fortran example is doing. Also indentation looks weird here.

Reply:
Done.